### PR TITLE
Adjusting wording and grammar.

### PR DESCRIPTION
**Pull request by Turbine2k5**

### CODE REVIEW

File: src/model/db/AbstractModel.ts [hunk 1]
```diff
@@ -10,8 +10,8 @@ export abstract class AbstractModel {
     };
 
     @PrimaryGeneratedColumn("increment")
-    @Name("id")
-    @Description("The id of this entry")
+    @Name("ID")
```
Comment (VictoriqueMoe) [inline src/model/db/AbstractModel.ts:13]: `@Name` can't be changed unless you update the JS along woth it, `@Name` are bound body paerams and are are camal case.

File: src/services/SubmissionConfirmationService.ts [hunk 2]
```diff
@@ -67,7 +67,7 @@ export class SubmissionConfirmationService implements OnInit {
     private sendConfirmationEmail(pendingEntry: PendingEntryConfirmationModel): Promise<string> {
         const baseUrl = process.env.BASE_URL;
         const confirmationUrl = `${baseUrl}/processSubmission?uid=${pendingEntry.confirmationUid}`;
-        const body = `Please click the link below to confirm your submission. This link will expire in 20 minutes\n${confirmationUrl}`;
+        const body = `Please click the link below to confirm your submission. This link will expire in 20 minutes. \n${confirmationUrl}`;
```
Comment (VictoriqueMoe) [inline src/services/SubmissionConfirmationService.ts:70]: remove the space between `.` and `/n`
```suggestion
        const body = `Please click the link below to confirm your submission. This link will expire in 20 minutes.\n${confirmationUrl}`;
```

**Review by VictoriqueMoe — CHANGES_REQUESTED** [review]

**Comment by VictoriqueMoe** [comment]
revert all `@name` defenitons

**Comment by Turbine2k5** [comment]
Issues fixed. Also adjusted some of the form files.

File: src/model/db/PendingEntryConfirmation.model.ts [hunk 3]
```diff
@@ -10,8 +10,8 @@ import {Description, Example, Format, Name, Required} from "@tsed/schema";
 })
 export class PendingEntryConfirmationModel extends AbstractModel {
 
-    @Name("confirmationUid")
-    @Description("Uid of the confirmation")
+    @Name("confirmationUID")
```
Comment (VictoriqueMoe) [inline src/model/db/PendingEntryConfirmation.model.ts:13]: revert

File: src/model/db/PendingEntryConfirmation.model.ts [hunk 4]
```diff
@@ -27,7 +27,7 @@ export class PendingEntryConfirmationModel extends AbstractModel {
     @Format("email")
     public submitterEmail: string;
 
-    @Name("submissionRoundId")
+    @Name("submissionRoundID")
```
Comment (VictoriqueMoe) [inline src/model/db/PendingEntryConfirmation.model.ts:30]: revert

**Review by VictoriqueMoe — CHANGES_REQUESTED** [review]

**Review by VictoriqueMoe — APPROVED** [review]

**Pull request merged**